src/sagemaker/script_uris.py

```diff
@@ -0,0 +1,50 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""Functions for generating S3 model script URIs for pre-built SageMaker models."""
+from __future__ import absolute_import
+
+import logging
+
+from sagemaker.jumpstart import utils as jumpstart_utils
+from sagemaker.jumpstart import constants as jumpstart_constants
+from sagemaker.jumpstart import artifacts
+
+logger = logging.getLogger(__name__)
+
+
+def retrieve(
+    region=jumpstart_constants.JUMPSTART_DEFAULT_REGION_NAME,
+    model_id=None,
+    model_version=None,
+    script_scope=None,
```

Comment:
same comment on this signature.

Reply:
nvm - I agree that keeping it consistent with the `retrieve` utility of the `image_uris` module makes sense.